platform/linux-generic/Makefile.am

```diff
@@ -210,6 +210,7 @@ __LIB__libodp_linux_la_SOURCES = \
 			   odp_parse.c \
 			   odp_pkt_queue.c \
 			   odp_pool.c \
+			   odp_pool_mem_src_ops.c \
```

Comment:
To be more consistent with current file naming convention (e.g. `odp_queue_if.c`, `odp_schedule_if.c`) this file could be `odp_pool_if.c`.